recipes/charls/all/test_package/CMakeLists.txt

```diff
@@ -2,12 +2,14 @@ cmake_minimum_required(VERSION 3.1.3)
 project(test_package)
 
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup()
+conan_basic_setup(TARGETS)
+
+find_package(charls CONFIG REQUIRED)
 
 add_executable(test_package_c test_package.c)
-target_link_libraries(test_package_c ${CONAN_LIBS})
+target_link_libraries(test_package_c charls::charls)
```

Comment:
official target is not namespaced I think: https://github.com/team-charls/charls/blob/3dd2e03a83a4d69825c984f7a7e8e319ecae1cd2/src/CMakeLists.txt#L114

Reply:
I have missed it.
I will fix later.

Reply:
I will try to fix it.
Cloud you check my modification?
I'm afraid I don't follow your comment.